### PR TITLE
Fixes#344 : Adds user guide link as info icon and restores space between radio and text

### LINKED ISSUE #344: Instead of "User Guide" add an ⓘ icon
Please add a question "ⓘ" icon on the top right of the app box instead of the field "User Input Guide
Check out our User Input Guide".

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #344 #345 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Please find [deployment link](https://lit-harbor-67991.herokuapp.com/)
- Adds user guide link as info icon 
- restores space between radio and text
image

Closes #344 #345

### CODE REVIEW

**Comment by ParthS007** [comment]
removing it just read Mario comment in previous PR :+1:

**Comment by harshit98** [comment]
Hi @ParthS007. 

Nice work! Can you provide demo link with your PR ? It will be easier for the mentors to review the PR fast.

**Comment by ParthS007** [comment]
@harshit98 @djmgit Please review

**Comment by ParthS007** [comment]
@djmgit It is due to priority issue when I am using it from style.css, the CSS property is not effective that's why I made another file.

**Comment by harshit98** [comment]
@ParthS007 I think @djmgit is right. For styling only one element there is no need of creating another `info.css`. Try to put in `style.css`.

**Comment by ParthS007** [comment]
@djmgit I have tried by putting style in style.css but it is not been effective. Csn you please help me as I have tried almost every way to do it.

**Comment by vaibhavsingh97** [comment]
@ParthS007 Wait, I will help 😄

**Comment by vaibhavsingh97** [comment]
@ParthS007 I tried to put the class in style and it worked perfectly, so can please check?

**Comment by ParthS007** [comment]
@djmgit @harshit98 @vaibhavsingh97 please review now. I have made all the suggested changes.

File: app/static/css/style.css [hunk 1]
```diff
@@ -5,6 +5,11 @@
     padding-bottom: 20px;
 }
 
+#in{
```
Comment (vaibhavsingh97) [inline app/static/css/style.css]: ID name is not meaningful, please use some meaningful ID name.

File: app/static/css/style.css [hunk 2]
```diff
@@ -5,6 +5,11 @@
...
+    text-indent: 3.5cm ! important;
+    font-size: 24px ! important;
```
Comment (vaibhavsingh97) [inline app/static/css/style.css]: Don't use important unitil you want to override. Please read more about it [here](https://stackoverflow.com/questions/3427766/should-i-avoid-using-important-in-css) and [here](https://css-tricks.com/when-using-important-is-the-right-choice/)

Comment (ParthS007) [reply app/static/css/style.css]: I want to override that's why I have used it

File: app/static/css/style.css (hunk 1 above)
Comment (ParthS007) [reply app/static/css/style.css]: I want to write info but it was there in default so I used in

**Comment by harshit98** [comment]
@vaibhavsingh97 Even I feel the same. `!important` should not be used. Please help @ParthS007 so that this PR gets merged. 👍

**Comment by vaibhavsingh97** [comment]
Yeah! @harshit98 Already guided him 👍

**Pull request merged**